### PR TITLE
Berlin - WDFT MAY2021 - Gergö Eperjesi

### PR DESCRIPTION
**Pull request by GergoEper**
- Started to create the header in HTML

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,9 +1,86 @@
...
+        </div>
+        <div>
+            <img class="npm-logo" src="images/npm-logo.png" alt="npm-logo"/>
+            <span class="search-span">
+                <span id="magnifying-glass-span">
+                <img class="magnifying-glass" src="images/magnifying-glass.png" alt="magnifying-glass"/>
+                </span> 
+                <input id="search-bar" type="search" placeholder="Search packages">
+                <button id="search-bar-btn" type="submit">Search</button>
+            </span>
```
Comment (annawaszak) [inline index.html:36]: I would recommend to replace that "span" with a "div"

File: index.html [hunk 2]
```diff
@@ -1,9 +1,86 @@
...
+              <li><a href="#"> Support </a></li>
+            </ul>
+          </nav>
+        </div>
+        <div>
+            <img class="npm-logo" src="images/npm-logo.png" alt="npm-logo"/>
+            <span class="search-span">
+                <span id="magnifying-glass-span">
+                <img class="magnifying-glass" src="images/magnifying-glass.png" alt="magnifying-glass"/>
+                </span>
```
Comment (annawaszak) [inline index.html:33]: I would recommend to replace that "span" with a "div"

File: index.html [hunk 3]
```diff
@@ -1,9 +1,86 @@
...
+              <li><a href="#"> Docs </a></li>
+              <li><a href="#"> Support </a></li>
+            </ul>
+          </nav>
+        </div>
+        <div>
+            <img class="npm-logo" src="images/npm-logo.png" alt="npm-logo"/>
+            <span class="search-span">
+                <span id="magnifying-glass-span">
+                <img class="magnifying-glass" src="images/magnifying-glass.png" alt="magnifying-glass"/>
```
Comment (annawaszak) [inline index.html:32]: Following the proper indentation, this img would be moved a bit more to the right.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+    align-items: center;
+    flex-direction: row;
+    background-color: #fff;
+    height: 400px;
+    padding-left: 2rem;
+    padding-right: 2rem;
+  }
+
+  #bottom-div {
+    box-sizing: border-box;
```
Comment (annawaszak) [inline style.css:198]: I would recommend to set a specific width for a box so they are all the same size. They could also use some margin.

File: style.css [hunk 5]
```diff
@@ -0,0 +1,231 @@
...
+    background-color: rgba(232,217,217,.3);
+    height: 600px;
+  }
+
+  #section-h1 {
+    box-sizing: border-box;
+    font-size: 5rem;
+    margin: 0.60em 0;
+    font-weight: 900;
+    box-sizing: border-box;
```
Comment (annawaszak) [inline style.css:113]: This is repeated. Also, I'm not sure what change does it make.

File: style.css [hunk 6]
```diff
@@ -0,0 +1,231 @@
...
+    align-items: center;
+    flex-direction: column;
+    background-color: rgba(232,217,217,.3);
+    height: 600px;
+  }
+
+  #section-h1 {
+    box-sizing: border-box;
+    font-size: 5rem;
+    margin: 0.60em 0;
```
Comment (annawaszak) [inline style.css:111]: If you want to manipulate the top margin only, I would delete this part. 
If you want to manipulate all of them, I would delete the "margin-top" from below and use this to do what you need.

**Review by annawaszak — COMMENTED** [review]
Hi Gergo,

It's nice to virtually meet you!

I have reviewed your NPM Clone lab and you did a really great job. There are only a few small improvements I would suggest - you can see them in the comments below.

If you have any follow-up questions, don't hesitate to respond to my comments.

Thanks,
Anna

**Comment by GergoEper** [comment]
Hi Anna,

Thanks for the feedback!:)
I will make the changes you suggested, I wasn't sure about using the "span"
either, I think I used it because they were using it on the original
website (I could find the old wesion on https://web.archive.org/).

Thank you!,
Gergö

Anna Waszak ***@***.***> ezt írta (időpont: 2021. máj. 26.,
Sze, 18:53):

**Pull request closed without merging**